Support for Python 3.10?

## Feature Request

Is this a New Feature, or a Proposed Change to existing Behavior? 
This is the continuity of existing feature.

## Description

Describe the new feature, or describe the modification to existing behavior, and provide context as necessary to help with understanding how the feature relates to a workflow or functionality.
I am the developer of [Shot Manager](https://github.com/ubisoft/shotmanager), a Blender add-on that provides many features to manipulate shots in the 3D space of Blender but also to import and export edits with other software. To do so I rely on OpenTimelineIO.
Thing is Blender is changing regularly (or irregularly I should say) the version of Python they support, this in order to stick to the latest version. An in March Blender 3.1 will be on Python 3.10.

Would it be possible to have a version of OTIO for Python 3.10? That would be of a great help!
Thank you

This is the error I get when installing opentimelineio under shotmanager addon with Blender 3.1 which is using python 3.10

`Collecting opentimelineio
  Using cached OpenTimelineIO-0.14.1.tar.gz (2.2 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Collecting pyaaf2~=1.4.0
  Using cached pyaaf2-1.4.0.tar.gz (88 kB)
Using legacy 'setup.py install' for pyaaf2, since package 'wheel' is not installed.
Building wheels for collected packages: opentimelineio
  Building wheel for opentimelineio (PEP 517) ... error
  ERROR: Command errored out with exit status 1:
   command: 'X:\development\blender-git\build_windows_Release_x64_vc16_Release\bin\Release\3.2\python\bin\python.exe' 'X:\development\blender-git\build_windows_Release_x64_vc16_Release\bin\Release\3.2\python\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py' build_wheel 'C:\Users\kreed\AppData\Local\Temp\tmpq27fd9bn'
       cwd: C:\Users\kreed\AppData\Local\Temp\pip-install-kg_31jge\opentimelineio_52e3b6b997dd47929d61cd32a8cba4f5
  Complete output (192 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.10
  creating build\lib.win-amd64-3.10\opentimelineio
  copying src\py-opentimelineio\opentimelineio\exceptions.py -> build\lib.win-amd64-3.10\opentimelineio
  copying src\py-opentimelineio\opentimelineio\hooks.py -> build\lib.win-amd64-3.10\opentimelineio
  copying src\py-opentimelineio\opentimelineio\media_linker.py -> build\lib.win-amd64-3.10\opentimelineio
  copying src\py-opentimelineio\opentimelineio\opentime.py -> build\lib.win-amd64-3.10\opentimelineio
  copying src\py-opentimelineio\opentimelineio\test_utils.py -> build\lib.win-amd64-3.10\opentimelineio
  copying src\py-opentimelineio\opentimelineio\url_utils.py -> build\lib.win-amd64-3.10\opentimelineio
  copying src\py-opentimelineio\opentimelineio\__init__.py -> build\lib.win-amd64-3.10\opentimelineio
  creating build\lib.win-amd64-3.10\opentimelineio\adapters
  copying src\py-opentimelineio\opentimelineio\adapters\adapter.py -> build\lib.win-amd64-3.10\opentimelineio\adapters
  copying src\py-opentimelineio\opentimelineio\adapters\cmx_3600.py -> build\lib.win-amd64-3.10\opentimelineio\adapters
  copying src\py-opentimelineio\opentimelineio\adapters\fcp_xml.py -> build\lib.win-amd64-3.10\opentimelineio\adapters
  copying src\py-opentimelineio\opentimelineio\adapters\file_bundle_utils.py -> build\lib.win-amd64-3.10\opentimelineio\adapters
  copying src\py-opentimelineio\opentimelineio\adapters\otiod.py -> build\lib.win-amd64-3.10\opentimelineio\adapters
  copying src\py-opentimelineio\opentimelineio\adapters\otioz.py -> build\lib.win-amd64-3.10\opentimelineio\adapters
  copying src\py-opentimelineio\opentimelineio\adapters\otio_json.py -> build\lib.win-amd64-3.10\opentimelineio\adapters
  copying src\py-opentimelineio\opentimelineio\adapters\svg.py -> build\lib.win-amd64-3.10\opentimelineio\adapters
  copying src\py-opentimelineio\opentimelineio\adapters\__init__.py -> build\lib.win-amd64-3.10\opentimelineio\adapters
  creating build\lib.win-amd64-3.10\opentimelineio\algorithms
  copying src\py-opentimelineio\opentimelineio\algorithms\filter.py -> build\lib.win-amd64-3.10\opentimelineio\algorithms
  copying src\py-opentimelineio\opentimelineio\algorithms\stack_algo.py -> build\lib.win-amd64-3.10\opentimelineio\algorithms
  copying src\py-opentimelineio\opentimelineio\algorithms\timeline_algo.py -> build\lib.win-amd64-3.10\opentimelineio\algorithms
  copying src\py-opentimelineio\opentimelineio\algorithms\track_algo.py -> build\lib.win-amd64-3.10\opentimelineio\algorithms
  copying src\py-opentimelineio\opentimelineio\algorithms\__init__.py -> build\lib.win-amd64-3.10\opentimelineio\algorithms
  creating build\lib.win-amd64-3.10\opentimelineio\console
  copying src\py-opentimelineio\opentimelineio\console\autogen_plugin_documentation.py -> build\lib.win-amd64-3.10\opentimelineio\console
  copying src\py-opentimelineio\opentimelineio\console\autogen_serialized_datamodel.py -> build\lib.win-amd64-3.10\opentimelineio\console
  copying src\py-opentimelineio\opentimelineio\console\console_utils.py -> build\lib.win-amd64-3.10\opentimelineio\console
  copying src\py-opentimelineio\opentimelineio\console\otiocat.py -> build\lib.win-amd64-3.10\opentimelineio\console
  copying src\py-opentimelineio\opentimelineio\console\otioconvert.py -> build\lib.win-amd64-3.10\opentimelineio\console
  copying src\py-opentimelineio\opentimelineio\console\otiopluginfo.py -> build\lib.win-amd64-3.10\opentimelineio\console
  copying src\py-opentimelineio\opentimelineio\console\otiostat.py -> build\lib.win-amd64-3.10\opentimelineio\console
  copying src\py-opentimelineio\opentimelineio\console\__init__.py -> build\lib.win-amd64-3.10\opentimelineio\console
  creating build\lib.win-amd64-3.10\opentimelineio\core
  copying src\py-opentimelineio\opentimelineio\core\composable.py -> build\lib.win-amd64-3.10\opentimelineio\core
  copying src\py-opentimelineio\opentimelineio\core\composition.py -> build\lib.win-amd64-3.10\opentimelineio\core
  copying src\py-opentimelineio\opentimelineio\core\item.py -> build\lib.win-amd64-3.10\opentimelineio\core
  copying src\py-opentimelineio\opentimelineio\core\mediaReference.py -> build\lib.win-amd64-3.10\opentimelineio\core
  copying src\py-opentimelineio\opentimelineio\core\_core_utils.py -> build\lib.win-amd64-3.10\opentimelineio\core
  copying src\py-opentimelineio\opentimelineio\core\__init__.py -> build\lib.win-amd64-3.10\opentimelineio\core
  creating build\lib.win-amd64-3.10\opentimelineio\plugins
  copying src\py-opentimelineio\opentimelineio\plugins\manifest.py -> build\lib.win-amd64-3.10\opentimelineio\plugins
  copying src\py-opentimelineio\opentimelineio\plugins\python_plugin.py -> build\lib.win-amd64-3.10\opentimelineio\plugins
  copying src\py-opentimelineio\opentimelineio\plugins\__init__.py -> build\lib.win-amd64-3.10\opentimelineio\plugins
  creating build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\clip.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\effect.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\external_reference.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\foo.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\generator_reference.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\image_sequence_reference.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\marker.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\schemadef.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\serializable_collection.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\stack.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\timeline.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\track.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\transition.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  copying src\py-opentimelineio\opentimelineio\schema\__init__.py -> build\lib.win-amd64-3.10\opentimelineio\schema
  creating build\lib.win-amd64-3.10\opentimelineio\schemadef
  copying src\py-opentimelineio\opentimelineio\schemadef\__init__.py -> build\lib.win-amd64-3.10\opentimelineio\schemadef
  creating build\lib.win-amd64-3.10\opentimelineview
  copying src\opentimelineview\console.py -> build\lib.win-amd64-3.10\opentimelineview
  copying src\opentimelineview\details_widget.py -> build\lib.win-amd64-3.10\opentimelineview
  copying src\opentimelineview\ruler_widget.py -> build\lib.win-amd64-3.10\opentimelineview
  copying src\opentimelineview\settings.py -> build\lib.win-amd64-3.10\opentimelineview
  copying src\opentimelineview\timeline_widget.py -> build\lib.win-amd64-3.10\opentimelineview
  copying src\opentimelineview\track_widgets.py -> build\lib.win-amd64-3.10\opentimelineview
  copying src\opentimelineview\__init__.py -> build\lib.win-amd64-3.10\opentimelineview
  creating build\lib.win-amd64-3.10\opentimelineio_contrib
  copying contrib\opentimelineio_contrib\__init__.py -> build\lib.win-amd64-3.10\opentimelineio_contrib
  creating build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\advanced_authoring_format.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\ale.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\burnins.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\extern_maya_sequencer.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\extern_rv.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\fcpx_xml.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\ffmpeg_burnins.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\hls_playlist.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\kdenlive.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\maya_sequencer.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\rv.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\xges.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  copying contrib\opentimelineio_contrib\adapters\__init__.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  creating build\lib.win-amd64-3.10\opentimelineio_contrib\adapters\aaf_adapter
  copying contrib\opentimelineio_contrib\adapters\aaf_adapter\aaf_writer.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters\aaf_adapter
  copying contrib\opentimelineio_contrib\adapters\aaf_adapter\__init__.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters\aaf_adapter
  running egg_info
  writing OpenTimelineIO.egg-info\PKG-INFO
  writing dependency_links to OpenTimelineIO.egg-info\dependency_links.txt
  writing entry points to OpenTimelineIO.egg-info\entry_points.txt
  writing requirements to OpenTimelineIO.egg-info\requires.txt
  writing top-level names to OpenTimelineIO.egg-info\top_level.txt
  reading manifest file 'OpenTimelineIO.egg-info\SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  warning: no files found matching 'CHANGELOG.md'
  no previously-included directories found matching '.github'
  no previously-included directories found matching 'docs'
  no previously-included directories found matching 'doxygen'
  warning: no previously-included files found matching '.gitmodules'
  warning: no previously-included files matching '*.git' found under directory 'src'
  warning: no previously-included files found matching '.readthedocs.yml'
  warning: no previously-included files found matching 'readthedocs-conda.yml'
  warning: no previously-included files found matching '.codecov.yml'
  warning: no previously-included files found matching '.gitlab-ci.yml'
  warning: no previously-included files found matching '.travis.yml'
  warning: no previously-included files found matching '*.pdf'
  warning: no previously-included files found matching 'CODE_OF_CONDUCT.md'
  warning: no previously-included files found matching 'CONTRIBUTING.md'
  warning: no previously-included files found matching 'CONTRIBUTORS.md'
  warning: no previously-included files found matching 'contrib\opentimelineio_contrib\adapters\Makefile'
  warning: no previously-included files found matching 'Makefile'
  warning: no previously-included files found matching '*\.DS_Store'
  warning: no previously-included files found matching '.clang-format'
  no previously-included directories found matching 'contrib\opentimelineio_contrib\adapters\tests'
  no previously-included directories found matching 'maintainers'
  no previously-included directories found matching 'tests'
  no previously-included directories found matching 'src\deps\pybind11\tools\clang'
  no previously-included directories found matching 'src\deps\rapidjson\thirdparty'
  adding license file 'LICENSE.txt'
  adding license file 'NOTICE.txt'
  writing manifest file 'OpenTimelineIO.egg-info\SOURCES.txt'
  copying src\py-opentimelineio\opentimelineio\adapters\builtin_adapters.plugin_manifest.json -> build\lib.win-amd64-3.10\opentimelineio\adapters
  creating build\lib.win-amd64-3.10\opentimelineio_contrib\application_plugins
  creating build\lib.win-amd64-3.10\opentimelineio_contrib\application_plugins\rv
  creating build\lib.win-amd64-3.10\opentimelineio_contrib\application_plugins\rv\example_otio_reader
  copying contrib\opentimelineio_contrib\application_plugins\rv\example_otio_reader\PACKAGE -> build\lib.win-amd64-3.10\opentimelineio_contrib\application_plugins\rv\example_otio_reader
  copying contrib\opentimelineio_contrib\application_plugins\rv\example_otio_reader\example_otio_reader_plugin.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\application_plugins\rv\example_otio_reader
  copying contrib\opentimelineio_contrib\application_plugins\rv\example_otio_reader\otio_reader.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\application_plugins\rv\example_otio_reader
  creating build\lib.win-amd64-3.10\opentimelineio_contrib\application_plugins\tests
  copying contrib\opentimelineio_contrib\application_plugins\tests\__init__.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\application_plugins\tests
  copying contrib\opentimelineio_contrib\application_plugins\tests\test_rv_reader.py -> build\lib.win-amd64-3.10\opentimelineio_contrib\application_plugins\tests
  copying contrib\opentimelineio_contrib\adapters\contrib_adapters.plugin_manifest.json -> build\lib.win-amd64-3.10\opentimelineio_contrib\adapters
  running build_ext
  running OTIO build_ext
  running cmake check
  Traceback (most recent call last):
    File "X:\development\blender-git\build_windows_Release_x64_vc16_Release\bin\Release\3.2\python\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 349, in <module>
      main()
    File "X:\development\blender-git\build_windows_Release_x64_vc16_Release\bin\Release\3.2\python\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 331, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "X:\development\blender-git\build_windows_Release_x64_vc16_Release\bin\Release\3.2\python\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 248, in build_wheel
      return _build_backend().build_wheel(wheel_directory, config_settings,
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\build_meta.py", line 244, in build_wheel
      return self._build_with_temp_dir(['bdist_wheel'], '.whl',
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\build_meta.py", line 229, in _build_with_temp_dir
      self.run_setup()
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\build_meta.py", line 174, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 264, in <module>
      setup(
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\__init__.py", line 155, in setup
      return distutils.core.setup(**attrs)
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\_distutils\core.py", line 148, in setup
      return run_commands(dist)
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\_distutils\core.py", line 163, in run_commands
      dist.run_commands()
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 967, in run_commands
      self.run_command(cmd)
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 986, in run_command
      cmd_obj.run()
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\wheel\bdist_wheel.py", line 299, in run
      self.run_command('build')
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\_distutils\cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 986, in run_command
      cmd_obj.run()
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\_distutils\command\build.py", line 135, in run
      self.run_command(cmd_name)
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\_distutils\cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "C:\Users\kreed\AppData\Local\Temp\pip-build-env-jwsg8rzb\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 986, in run_command
      cmd_obj.run()
    File "setup.py", line 57, in run
      self.build()
    File "setup.py", line 71, in build
      self.cmake_preflight_check()
    File "setup.py", line 140, in cmake_preflight_check
      raise RuntimeError(stderr.strip())
  RuntimeError: CMake Error at C:/Users/kreed/AppData/Local/Temp/pip-build-env-jwsg8rzb/overlay/Lib/site-packages/cmake/data/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
    Could NOT find Python (missing: Python_LIBRARIES Python_INCLUDE_DIRS
    Interpreter Development.Module)
  Call Stack (most recent call first):
    C:/Users/kreed/AppData/Local/Temp/pip-build-env-jwsg8rzb/overlay/Lib/site-packages/cmake/data/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
    C:/Users/kreed/AppData/Local/Temp/pip-build-env-jwsg8rzb/overlay/Lib/site-packages/cmake/data/share/cmake-3.22/Modules/FindPython.cmake:561 (find_package_handle_standard_args)
    CMakeLists.txt:49 (find_package)
  ----------------------------------------
  ERROR: Failed building wheel for opentimelineio
Failed to build opentimelineio
ERROR: Could not build wheels for opentimelineio which use PEP 517 and cannot be installed directly
WARNING: You are using pip version 21.2.4; however, version 22.0.4 is available.`

--- a/.github/workflows/python-package.yml
+++ b/.github/workflows/python-package.yml
@@ -83,7 +83,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['2.7', '3.7', '3.8', '3.9']
+        python-version: ['2.7', '3.7', '3.8', '3.9', '3.10']
 
     env:
       OTIO_CXX_COVERAGE_BUILD: ON
@@ -133,7 +133,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-build: [cp27*, cp37*, cp38*, cp39*]
+        python-build: ['cp27*', 'cp37*', 'cp38*', 'cp39*', 'cp310*']
     steps:
       - uses: actions/checkout@v3
 
@@ -153,12 +153,25 @@ jobs:
           echo "DISTUTILS_USE_SDK=1" >> $GITHUB_ENV
           echo "MSSdk=1" >> $GITHUB_ENV
 
-      - name: Build wheels
+      - name: Build wheels (Python 2.7)
+        if: matrix.python-build == 'cp27*'
+        # cibuildwheel 1.12.0 is the last release that supported Python 2.7.
         uses: pypa/cibuildwheel@v1.12.0
         with:
           output-dir: wheelhouse
         env:
           CIBW_BUILD: ${{ matrix.python-build }}
+
+      - name: Build wheels (Python 3)
+        uses: pypa/cibuildwheel@v2.3.1
+        if: matrix.python-build != 'cp27*'
+        with:
+          output-dir: wheelhouse
+        env:
+          CIBW_BUILD: ${{ matrix.python-build }}
+          CIBW_SKIP: '*musllinux*'
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2010
+          CIBW_MANYLINUX_I686_IMAGE: manylinux2010
 
       - uses: actions/upload-artifact@v2
         with:

--- a/README.md
+++ b/README.md
@@ -4,7 +4,7 @@ OpenTimelineIO
 ==============
 
 [![Supported VFX Platform Versions](https://img.shields.io/badge/vfx%20platform-2018--2021-lightgrey.svg)](http://www.vfxplatform.com/)
-![Supported Versions](https://img.shields.io/badge/python-2.7%2C%203.7%2C%203.8%2C%203.9-blue)
+![Supported Versions](https://img.shields.io/badge/python-2.7%2C%203.7%2C%203.8%2C%203.9%2C%203.10-blue)
 [![Build Status](https://github.com/PixarAnimationStudios/OpenTimelineIO/actions/workflows/python-package.yml/badge.svg)](https://github.com/PixarAnimationStudios/OpenTimelineIO/actions/workflows/python-package.yml)
 [![codecov](https://codecov.io/gh/PixarAnimationStudios/OpenTimelineIO/branch/main/graph/badge.svg)](https://codecov.io/gh/PixarAnimationStudios/OpenTimelineIO)
 [![docs](https://readthedocs.org/projects/opentimelineio/badge/?version=latest)](https://opentimelineio.readthedocs.io/en/latest/index.html)
@@ -52,8 +52,7 @@ Supported VFX Platforms
 -----------------
 The current release supports:
 - VFX platform 2021, 2020, 2019, 2018
-- Python 2.7 - 3.9
-- Notice that Python 2.7 is deprecated & we plan to drop it in OTIO release 0.15
+- Python 2.7 - 3.10
 
 For more information on our vfxplatform support policy: [Contribution Guidelines Documentation Page](https://opentimelineio.readthedocs.io/en/latest/tutorials/contributing.html)
 For more information on the vfxplatform: [VFX Platform Homepage](https://vfxplatform.com)

--- a/setup.py
+++ b/setup.py
@@ -290,6 +290,7 @@ setup(
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
         'Operating System :: OS Independent',
         'Natural Language :: English',
     ],

